scripts/generate_screenshots/apply_variant.sh

```
@@ -0,0 +1,75 @@
+# $ ./scripts/generate_screenshots/apply_variant.sh <VARIANT_NAME>
+#
+# 引数:
+#   $1: VARIANT_NAME - 適用するバリアント名
+#       有効な値: ink (バリアントの一覧は appstore_screenshot_env.sh の get_variant_name が正)
+#
+# 例:
+#   $ ./scripts/generate_screenshots/apply_variant.sh ink
+#
+set -euo pipefail
+
+SCRIPT_DIR="$(cd `dirname $0` && pwd -P)"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)